packages/Molecules/Form/fields/fieldRplchecklist.vue

```diff
@@ -62,6 +62,13 @@ export default {
       labelText: ''
     }
   },
+  mounted () {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', this.updateSize)
+      this.updateSize()
+    }
+    this.setCheckedValues()
+  },
```

Comment:
I moved this because I didn't see it all the way down the bottom and ended up rewriting it. We should probably add linter rule for ordering instance methods. I'd recommend we follow vue style guide for this https://vuejs.org/v2/style-guide/#Component-instance-options-order-recommended.